### PR TITLE
Noop Razor Sdk for non C# projects

### PR DESCRIPTION
**Pull request by ajaybhargavb**
https://github.com/aspnet/Razor/issues/2188

- Added two integrations tests and an F# test app

### CODE REVIEW

File: test/testapps/SimpleMvcFSharp/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+﻿@{
```
Comment (pranavkm) [inline test/testapps/SimpleMvcFSharp/Views/Home/Index.cshtml]: Could you simplify this to have one or two small views? We don't need the full gamut of an F# project checked in for our tests

File: test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs [hunk 2]
```diff
@@ -455,6 +455,20 @@ public Task Build_SimpleMvc_UsingDesktopMSBuildAndWithoutBuildServer_CanBuildSuc
...
+        public async Task Build_SimpleMvcFSharp_NoopsWithoutFailing()
+        {
+            var result = await DotnetMSBuild("Build");
+
+            Assert.BuildPassed(result);
+
+            Assert.FileExists(result, OutputPath, "SimpleMvcFSharp.dll");
+            Assert.FileExists(result, OutputPath, "SimpleMvcFSharp.pdb");
+            Assert.FileExists(result, IntermediateOutputPath, "SimpleMvcFSharp.dll");
+            Assert.FileExists(result, IntermediateOutputPath, "SimpleMvcFSharp.pdb");
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs]: FileDoesNotExist("SimpleMvcFSharp.Views.dll")

**Comment by ajaybhargavb** [comment]
🆙 📅

File: test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs (hunk 2 above)
Comment (ajaybhargavb) [reply test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs]: Added.

**Pull request merged**